src/content/guides/getting-started.md

```diff
@@ -281,9 +281,8 @@ __package.json__
     "version": "1.0.0",
     "description": "",
     "scripts": {
--      "test": "echo \"Error: no test specified\" && exit 1"
-+      "test": "echo \"Error: no test specified\" && exit 1",
-+      "build": "webpack"
++      "build": "webpack",
+       "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
The two lines are the same except for the tail comma.

Reply:
Please revert this change. If you're going to change this one, you need to update all the following guides listing `package.json`.